rts/Sim/Misc/ExitOnlyMap.h

```diff
@@ -0,0 +1,50 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
+#ifndef EXIT_ONLY_MAP_H
+#define EXIT_ONLY_MAP_H
+
+// #include <array>
+#include <cstdint>
+#include <vector>
+
+#include "System/Log/ILog.h"
+
+class ExitOnlyMap {
+public:
+    static constexpr int resolution = 2;
+    static constexpr int sectorWidth = 16;
```

Comment:
Where do the 16 and 2 come from? Is it because a 1x1 building footprint is 16x16 elmos, which is 2x2 game squares? If that is the case then consider `#include <Sim/Misc/GlobalConstants.h>` and
```suggestion
    static constexpr int resolution = SPRING_FOOTPRINT_SCALE;
    static constexpr int sectorWidth = BUILD_SQUARE_SIZE;
```

Reply:
`resolution` is correct yes, I'll use the constant. `sectorWidth` isn't needed anymore - so I'll remove that.